### DISCUSSION TITLE
urban land allocation

### DISCUSSION THREAD

**Discussion by mxq517** [JGCRI/gcam-core#457] [Q&A]
I'm a beginner in GCAM, the output show that urban area is unchanged over time, in which xml file can I find the change in urban area I want to set? I have cheked all the xml about land，land 1,2,3,4,5, but I can't find the ideas. Thank you all.

**Comment by pkyle** [comment]
Unfortunately there still is no way to specify an exogenous pathway of urban land expansion in GCAM. Urban land is treated the same way as desert or tundra, in that the quantity specified in 2015 is simply carried forward into the future. I believe those quantities are assigned in `land_input_1.xml`, but again it is not user-configurable going into future periods.
